.gitignore

```diff
@@ -12,3 +12,4 @@ releases/*.tar.gz
 releases/*.tar.gz.sha256sum
 _output/
 .vagrant/
+.idea/
```

Comment:
nit: Generally it's preferred to exclude personal IDE configs or at least split them out into separate PRs vs. including them in bugfixes / feature additions